doc/spire_server.md

```diff
@@ -34,6 +34,7 @@ This document is a configuration reference for SPIRE Server. It includes informa
 | Notifier   | [k8sbundle](/doc/plugin_server_notifier_k8sbundle.md) | A notifier that pushes the latest trust bundle contents into a Kubernetes ConfigMap. |
 | UpstreamCA | [disk](/doc/plugin_server_upstreamca_disk.md) | Uses a CA loaded from disk to sign SPIRE server intermediate certificates. |
 | UpstreamCA | [awssecret](/doc/plugin_server_upstreamca_awssecret.md) | Uses a CA loaded from AWS SecretsManager to sign SPIRE server intermediate certificates. |
+| UpstreamCA | [spire](/doc/plugin_server_upstreamca_spire.md) | Uses a CA loaded from upstream SPIRE server in the same trust domain to sign SPIRE server intermediate certificates. |
```

Comment:
Similar comment here: `Uses an upstream SPIRE server in the same trust domain to obtain intermediate signing certificates for SPIRE server`